### PR TITLE
Home screen overhaul / InAppBrowser styling / Drawer Icons

### PR DESCRIPTION
**Pull request by CalebMcOlin**
Developed with:
- @dagmawidemissie17
- @zmagar

## Description
- Overhauled the main menu. 
  - New buttons linking to the cascade website
  - Added background-color
  - Styling for all buttons
- Added Icons to the Drawer navigation
- Updated all InAppBrowsers to have the same styles
- Added Company logo to header action bar

## To Test
1. Run the app
2. Test every button and ensure all links work
3. Rotate the device and make sure formatting still works
4. Test all screens

### CODE REVIEW

**Review by dagmawidemissie17 — APPROVED** [review]
Looking Good! 👌

File: components/routes/Drawers.js [hunk 1]
```diff
@@ -11,12 +13,34 @@ const Drawer = createDrawerNavigator();
 export const DefaultDrawer = () => {
   return (
     <Drawer.Navigator>
-      <Drawer.Group screenOptions={() => ({
+      <Drawer.Group screenOptions={({ route }) => ({
+        headerShown: true,
+        drawerIcon: ({ color, size }) => {
+          let iconName;
+          console.log(route.name)
```
Comment (dagmawidemissie17) [inline components/routes/Drawers.js:20]: We forgot to remove this.

Comment (CalebMcOlin) [reply components/routes/Drawers.js]: Good catch. Just removed it.

**Review by dagmawidemissie17 — APPROVED** [review]
LGTM

**Review by zmagar — APPROVED** [review]

**Pull request merged**